CDCgov/prime-reportstream
# 7776 - Create Events to track success of Public File Validation

This PR ...

Tracks how the File Validator is used as well as captures errors and warnings.

Test Steps:
1. Log into RS
2. Select an Federal Organization
3. Navigate to Admin -> Validate 
4. Upload a .csv or HL7 file
5. Check the logs in Azure to verify that the following were recorded in the customEvent table:
warningCount
errorCount
fileType
selected schema
sender

### Testing
